$OpenBSD: patch-interfaces_time_ui,v 1.3 2010/09/30 07:38:53 ajacoutot Exp $

XXX Do not display NTP configuration/synchronisation, it requires
"services" support.

--- interfaces/time.ui.orig	Thu Jul  1 12:29:55 2010
+++ interfaces/time.ui	Tue Sep 28 17:17:46 2010
@@ -364,7 +364,7 @@
         </child>
         <child>
           <object class="GtkLabel" id="label120">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
             <property name="xalign">0</property>
             <property name="label" translatable="yes">_Configuration:</property>
             <property name="use_underline">True</property>
@@ -378,7 +378,7 @@
         </child>
         <child>
           <object class="GtkComboBox" id="configuration_options">
-            <property name="visible">True</property>
+            <property name="visible">False</property>
           </object>
           <packing>
             <property name="left_attach">1</property>
@@ -589,44 +589,6 @@
               </object>
               <packing>
                 <property name="position">4</property>
-              </packing>
-            </child>
-            <child>
-              <object class="GtkButton" id="update_time">
-                <property name="visible">True</property>
-                <property name="can_focus">True</property>
-                <property name="receives_default">False</property>
-                <property name="tooltip_text" translatable="yes">Synchronize now</property>
-                <child>
-                  <object class="GtkAlignment" id="alignment13">
-                    <property name="visible">True</property>
-                    <property name="xscale">0</property>
-                    <property name="yscale">0</property>
-                    <child>
-                      <object class="GtkHBox" id="hbox56">
-                        <property name="visible">True</property>
-                        <property name="spacing">2</property>
-                        <child>
-                          <object class="GtkImage" id="image16">
-                            <property name="visible">True</property>
-                            <property name="stock">gtk-refresh</property>
-                            <property name="icon-size">1</property>
-                          </object>
-                          <packing>
-                            <property name="expand">False</property>
-                            <property name="fill">False</property>
-                            <property name="position">0</property>
-                          </packing>
-                        </child>
-                      </object>
-                    </child>
-                  </object>
-                </child>
-              </object>
-              <packing>
-                <property name="expand">False</property>
-                <property name="fill">False</property>
-                <property name="position">5</property>
               </packing>
             </child>
           </object>
